packages/fury-adapter-swagger/lib/json-schema.js

```diff
@@ -93,6 +95,20 @@ const pathHasCircularReference = (paths, path, reference) => {
 };
 
 const dereference = (example, root, paths, path) => {
+  // We shouldn't even be dereferencing examples, but given how swagger-parser
+  // works it had been doing this from the start (which was caught later).
+  //
+  // See https://github.com/apiaryio/api-elements.js/issues/220
+  //
+  // At thsi point, changing that behaviour would be a significant breaking
+  // change and it will affect some of our larger users. Not to mention that
+  // swagger-parser will still dereference the examples in cases where our code
+  // path doesn't, it won't be easy to solve.
+  //
+  // The below code attemps to dereference an example, but if we can't we
+  // will just return the example (possibly a "reference object") to be
+  // the example value.
```

Comment:
I want to point out, the information in this comment is not new. I think it is worth clarifying what the function does and why. In the past there are also cases where invalid reference would be treated as example value, this PR just adds one additional case.